tensorflow/training/docker/2.6/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,198 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+ARG PYTHON=python3.8
+ARG PYTHON_VERSION=3.8.2
+
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG OPENSSL_VERSION=1.1.1g
```

Comment:
Please do not install openssl from source, as Ubuntu 20.04 has no issues with installing openssl from the version published by Canonical. Instead, please use the same method as https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.4/py3/Dockerfile.cpu#L49